### DISCUSSION TITLE
How to move (set up) \.chia folder to larger HDD?

### DISCUSSION THREAD

**Discussion by JOuAgent** [Chia-Network/chia-blockchain#8101] [General]
Hi,

soon I do not have enough space on my c:\ folder - max. size is 60 GB, now free only 3 GB.

How can I change (where to set up) c:\users\~\.chia to move for example to d:\~\.chia ???

Thank you for your help.

![chia-space](https://user-images.githubusercontent.com/89073867/129703455-66904956-524e-4171-845f-ac3eb58088ed.png)

**Comment by JOuAgent** [comment]
Solved - thank you to pyl: it is not the easiest thing to do on windows - but it is possible to set a "system env var" called CHIA_ROOT to point at another drive, for instance setting it to drive G:\chia - and copy the data from your users folder, ~/.chia to that folder, and then open up chia, it should start working out of that directory instead
https://www.alphr.com/environment-variables-windows-10/

**Comment by BrandtH22** [comment]
@JOuAgent , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
